exampleScreenCapturer/src/main/java/com/twilio/video/examples/screencapturer/ScreenCapturerService.java

```diff
@@ -0,0 +1,146 @@
+package com.twilio.video.examples.screencapturer;
+
+import android.annotation.TargetApi;
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.Service;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.Binder;
+import android.os.IBinder;
+import android.support.v4.app.NotificationCompat;
+
+
+class ScreenCapturerManager {
+    private ScreenCapturerService mService;
+    private Context mContext;
+    private State currentState = State.UNBIND_SERVICE;
+
+    /** Defines callbacks for service binding, passed to bindService() */
+    private ServiceConnection connection = new ServiceConnection() {
+
+        @Override
+        public void onServiceConnected(ComponentName className,
+                                       IBinder service) {
+            // We've bound to ScreenCapturerService, cast the IBinder and get ScreenCapturerService instance
+            ScreenCapturerService.LocalBinder binder = (ScreenCapturerService.LocalBinder) service;
+            mService = binder.getService();
+            currentState = State.BIND_SERVICE;
+        }
+
+        @Override
+        public void onServiceDisconnected(ComponentName arg0) {
+        }
+    };
+
+    /**
+     * An enum describing the possible states of a ScreenCapturerManager.
+     */
+    public enum State {
+        BIND_SERVICE,
+        START_FOREGROUND,
+        END_FOREGROUND,
+        UNBIND_SERVICE
+    }
+
+    ScreenCapturerManager(Context context) {
```

Comment:
One observation here is that the ScreenCapturerManager and Service will be used the same for every API level. What are your thoughts on making the ScreenCapturerManager API only used on Android Q and up? We can do this at the activity level. This ensures we don't have to debug Service management issues on API levels where it is not required to capture the screen.